### DISCUSSION TITLE
whl is not a supported wheel on this platform

### DISCUSSION THREAD

**Discussion by hzhv** [carla-simulator/carla#5053] [Q&A] [answered]
Hi all! 

When I try to install the client of Carla 0.9.12 through wheel file, the following was shown:

**"ERROR: carla-0.9.12-cp37-cp37m-manylinux_2_27_x86_32.whl is not a supported wheel on this platfo**rm."

Also, my pip version is as follows:

Name: pip
**Version: 21.3.1**
Summary: The PyPA recommended tool for installing Python packages.
Home-page: https://pip.pypa.io/
Author: The pip developers
Author-email: distutils-sig@python.org

I would like to know how to figure this out! Thank you all in advance!!!!

**Comment by JohnnyZZhao** [comment]
Hi, I am facing similar issues on Windows, can someone help explain?
"**ERROR: carla-0.9.13-cp37-cp37m-win_amd64.whl is not a supported wheel on this platform.**"

pip 22.3.1 (python 3.10)

**Reply by JohnnyZZhao** [reply, marked as answer]
My issue is resolved. It is Python 3.10 by default on my Windows. After I switch to Python 3.7 as the file name suggests, it works :)

**Comment by CMakey** [comment]
in addition, if anyone is use virtual env, take a look at u python version,  which in env is the wrong version
